src/web_app/js/sdputils.js

```diff
@@ -202,6 +202,80 @@ function maybeSetVideoSendInitialBitRate(sdp, params) {
   return sdp;
 }
 
+function removePayloadTypeFromMline(mLine, payloadType) {
+  mLine = mLine.split(' ');
+  for (var i = 0; i < mLine.length; ++i) {
+    if (mLine[i] === payloadType.toString()) {
+      mLine.splice(i, 1);
+    }
+  }
+  return mLine.join(' ');
+
+}
+
+function removeCodecByName(sdpLines, codec) {
+  var index = findLine(sdpLines, 'a=rtpmap', codec);
+  if (index === null) {
+    return sdpLines;
+  }
+  var payloadType = getCodecPayloadTypeFromLine(sdpLines[index]);
+  sdpLines.splice(index, 1);
+
+  // Search for the video m= line and remove the codec.
+  var mLineIndex = findLine(sdpLines, 'm=', 'video');
+  if (mLineIndex === null) {
+    return sdpLines;
+  }
+  sdpLines[mLineIndex] = removePayloadTypeFromMline(sdpLines[mLineIndex],
+    payloadType);
+  return sdpLines;
+}
+
+function removeCodecByPayloadType(sdpLines, payloadType) {
+  var index = findLine(sdpLines, 'a=rtpmap', payloadType.toString());
+  if (index === null) {
+    return sdpLines;
+  }
+  sdpLines.splice(index, 1);
+
+  // Search for the video m= line and remove the codec.
+  var mLineIndex = findLine(sdpLines, 'm=', 'video');
+  if (mLineIndex === null) {
+    return sdpLines;
+  }
+  sdpLines[mLineIndex] = removePayloadTypeFromMline(sdpLines[mLineIndex],
+    payloadType);
+  return sdpLines;
+}
+
+function maybeRemoveVideoFec(sdp, params) {
```

Comment:
add maybeRemoveVideoFec to https://github.com/webrtc/apprtc/pull/302/files#diff-6ff1b268bd04bd496d408c9ec7eb462eR12